index.bs

```diff
@@ -542,6 +542,19 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 			* Custom format [=string/starts with=] `"web "`("web" followed by U+0020 SPACE) prefix
 				and suffix (after stripping out `"web "`) passes the [=parsing a MIME type=] check.
 
+	<h3 id="unsanitized-data-types-x"><dfn>Unsanitized data types</dfn></h3>
+
+		The implementation MAY recognize the native OS clipboard format description
+		for the following data types, to be able to populate the
+		{{ClipboardItem}} with the correct [=/MIME type=], and
+		set the correct data format on the OS clipboard in response to copy and cut
```

Comment:
Is it right to say in response to "copy and cut events" given that this doesn't apply to the `DataTransfer` or `execCommand` APIs?

Reply:
Removed.